project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala

```diff
@@ -37,7 +37,12 @@ object Docker {
       label("CLIO_VERSION", version.value)
       expose(8080)
       add(artifact, artifactTargetPath)
-      entryPoint("java", "-jar", artifactTargetPath)
+      /*
```

Comment:
We could look for more specific env vars here and hard-code how they're used as java options, but we'd still need to use a shell entry point so that they'd expand properly at runtime. On the other hand we could look for env vars / build args at build time, but then our Docker images would have configuration baked-in.